CCC-Travel-Expense-Policy.md

```diff
@@ -0,0 +1,25 @@
+## Confidential Computing Consortium Travel Expense Policy
+
+
+The Confidential Computing Consortium’s Travel Budgets are intended to enable CCC’s project developers and community members to attend events that they would otherwise be unable to attend due to lack of funding. If you work for a company that has the ability to fund your travel, we ask that you seek funding from your company first, to save funds for those who need it. The goal is to try to use the CCC funds to assist as many people as possible.
+
+In order to maximize spend, travel fund assistance may only be used for:
+
+* Coach airfare tickets 
+* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Ground transportation to/from the airport
+
+**NOTE:** Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc. Generally, registration can be reimbursed if the attendee is speaking on behalf of the CCC or staffing a CCC booth at a sponsored event.
```

Comment:
is speaking on behavior of a Technical Project considered to be speaking on behalf of the CCC?